status/flowcells.py

```diff
@@ -244,7 +255,13 @@ class FlowcellLinksDataHandler(SafeHandler):
     def get(self, flowcell):
         self.set_header("Content-type", "application/json")
         try:
-            p = lims.get_containers(name=flowcell[8:])[0]
```

Comment:
It is used in two different classes, so the only way would be to define a
method outside of the class and use it, but I'm not sure it would be much
better.
I like to be able to figure out what 20 lines of code do wothout having to
scroll at the top/bottom of the file every two lines :P


On Mon, Apr 27, 2015 at 12:01 PM, Francesco <notifications@github.com>
wrote:

> In status/flowcells.py
> <https://github.com/SciLifeLab/genomics-status/pull/197#discussion_r29134269>
> :
>
> > @@ -244,7 +255,13 @@ class FlowcellLinksDataHandler(SafeHandler):
> >      def get(self, flowcell):
> >          self.set_header("Content-type", "application/json")
> >          try:
> > -            p = lims.get_containers(name=flowcell[8:])[0]
>
> @Galithil <https://github.com/Galithil> know you will hate me for this
> comment... but there is no way to abstract this check so we avoid this code
> repetition? would be much better
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SciLifeLab/genomics-status/pull/197/files#r29134269>.
>